{"neuraltalk/eval_sentence_predictions.py": {"error": "SyntaxError at line: 92"}, "neuraltalk/predict_on_images.py": {"loc": 109, "lloc": 73, "sloc": 68, "comments": 16, "multi": 12, "blank": 18, "single_comments": 11}, "neuraltalk/py_caffe_feat_extract.py": {"loc": 262, "lloc": 161, "sloc": 141, "comments": 19, "multi": 51, "blank": 66, "single_comments": 4}, "neuraltalk/driver.py": {"loc": 315, "lloc": 230, "sloc": 226, "comments": 77, "multi": 0, "blank": 38, "single_comments": 51}, "neuraltalk/imagernn/imagernn_utils.py": {"loc": 40, "lloc": 30, "sloc": 28, "comments": 5, "multi": 5, "blank": 5, "single_comments": 2}, "neuraltalk/imagernn/generic_batch_generator.py": {"loc": 153, "lloc": 110, "sloc": 100, "comments": 35, "multi": 5, "blank": 23, "single_comments": 25}, "neuraltalk/imagernn/solver.py": {"error": "SyntaxError at line: 105"}, "neuraltalk/imagernn/utils.py": {"loc": 26, "lloc": 22, "sloc": 19, "comments": 3, "multi": 0, "blank": 4, "single_comments": 3}, "neuraltalk/imagernn/rnn_generator.py": {"loc": 280, "lloc": 202, "sloc": 195, "comments": 63, "multi": 11, "blank": 44, "single_comments": 30}, "neuraltalk/imagernn/data_provider.py": {"loc": 116, "lloc": 90, "sloc": 80, "comments": 17, "multi": 0, "blank": 18, "single_comments": 18}, "neuraltalk/imagernn/lstm_generator.py": {"loc": 298, "lloc": 237, "sloc": 199, "comments": 73, "multi": 17, "blank": 39, "single_comments": 43}, "neuraltalk/imagernn/__init__.py": {"loc": 0, "lloc": 0, "sloc": 0, "comments": 0, "multi": 0, "blank": 0, "single_comments": 0}, "neuraltalk/python_features/extract_features.py": {"loc": 121, "lloc": 75, "sloc": 73, "comments": 7, "multi": 12, "blank": 29, "single_comments": 7}}
